nmigen_boards/fomu_hacker.py

```diff
@@ -0,0 +1,58 @@
+import os
+import subprocess
+
+from nmigen.build import *
+from nmigen.vendor.lattice_ice40 import *
+from .dev import *
+
+__all__ = ["FomuHackerPlatform"]
+
+
+class FomuHackerPlatform(LatticeICE40Platform):
+    device = "iCE40UP5K"
+    package = "UWG30"
+    default_clk = "clk48"
+
+    resources = [
+        Resource("clk48", 0,
+            Pins("F5", dir="i"),
+            Clock(48e6),
+            Attrs(GLOBAL="1", IO_STANDARD="SB_LVCMOS33"),
+        ),
+
+        Resource("user_led", 0, PinsN("A5", dir="o"), Attrs(IO_STANDARD="SB_LVCMOS33")),
+        Resource("rgb_led", 0,
+            Subsignal("r", PinsN("C5")),
+            Subsignal("g", PinsN("B5")),
+            Subsignal("b", PinsN("A5")),
+            Attrs(IO_STANDARD="SB_LVCMOS33"),
+        ),
+
+        Resource("usb", 0,
+            Subsignal("d_p", Pins("A4")),
+            Subsignal("d_n", Pins("A2")),
+            Subsignal("pullup", Pins("D5")),
+            Attrs(IO_STANDARD="SB_LVCMOS33"),
+        ),
+
+        *SPIFlashResources(0,
+            cs="C1",
+            clk="D1",
+            mosi="F1",
+            miso="E1",
+            attrs=Attrs(IO_STANDARD="SB_LVCMOS33"),
+        ),
+    ]
+
+    connectors = [Connector("touch_pins", 0, "F4 E5 E4 F2")]
```

Comment:
Why is this a connector?

Reply:
This is due to the possibility to use those as gpio pins. It is a kind of weird design ([see schematic](https://github.com/im-tomu/fomu-hardware/blob/master/hacker/releases/v0.0-19-g154fecc/schematic.pdf)). According to @xobs, they can be used as resistive touch sensors (with internal pulldown and a 3v3 plane on top of the board) but it seems to be fairly untested.

Reply:
I don't see them connected anywhere on schematic?

Reply:
Yes, they are just exposed pads on the PCB

Reply:
OK. It sounds like they should really be 4 connectors, `Connector("pin", 0, "XX")` and so on.